File: grazi/src/test/resources/ide/language/js/StringLiteral.js


var oneTypo = "It is <warning>friend</warning> of human";
var oneSpellcheckTypo = "It is <warning>frend</warning> of human";
var fewTypos = "It <warning>are</warning> working for <warning>much</warning> warnings";
var ignoreTemplate = "It is ${fewTypos} friend";
var notIgnoreOtherMistakes = "It is <warning>friend</warning>. <warning>But</warning> I have a ${1} here";


var oneTypo = 'It is <warning>friend</warning> of human';
var oneSpellcheckTypo = 'It is <warning>frend</warning> of human';
var fewTypos = 'It <warning>are</warning> working for <warning>much</warning> warnings';
var ignoreTemplate = 'It is ${fewTypos} friend';
var notIgnoreOtherMistakes = 'It is <warning>friend</warning>. <warning>But</warning> I have a ${1} here';


console.log("It is <warning>friend</warning> of human");
console.log("It is <warning>frend</warning> of human");
console.log("It <warning>are</warning> working for <warning>much</warning> warnings");
console.log("It is ${fewTypos} friend");
console.log("It is <warning>friend</warning>. <warning>But</warning> I have a ${1} here");

File: grazi/src/test/resources/ide/language/rust/StringLiteral.rs
#![crate_name = "doc"]

pub struct Person {
    name: String,
}

impl Person {
    pub fn new(name: &str) -> Person {
        Person {
            name: name.to_string(),
        }
    }

    pub fn hello(& self) {
        println!("Hello, {}!", self.name);
    }
}

fn main() {
    let john = Person::new("John");

    let oneTypo = "It is <warning>friend</warning> of human";
    let oneSpellcheckTypo = "It is <warning>frend</warning> of human";
    let fewTypos = "It <warning>are</warning> working for <warning>much</warning> warnings";
    let ignoreTemplate = "It is {} friend";
    let notIgnoreOtherMistakes = "It is <warning>friend</warning>. <warning>But</warning> I have a {} here";

    println!("It is <warning>friend</warning> of human");
    println!("It is <warning>frend</warning> of human");
    println!("It <warning>are</warning> working for <warning>much</warning> warnings");
    println!("It is {} friend", john.name);
    println!("It is <warning>friend</warning>. <warning>But</warning> I have a {} here", john.name);

    john.hello();
}
